src/api-mocks/submissions.js

```diff
@@ -136,3 +136,14 @@ export const mockSubmissionProcessingStatusErrorGet = rest.get(
       })
     )
 );
+
+export const mockSubmissionComplete = rest.post(
+  `${BASE_URL}submissions/:uuid/_complete`,
+  (req, res, ctx) =>
+    res(
+      ctx.json({
+        form: `${BASE_URL}forms/mock`,
+        formUrl: `http://example.com/mock`,
+      })
```

Comment:
looking a the API docs (on `/api/v2/docs/#tag/submissions/operation/submissions__complete_create`), the response of this endpoint is:

```json
{
  "statusUrl": "http://example.com"
}
```